src/rcc.rs

```diff
@@ -280,7 +281,7 @@ impl CFGR {
         assert!(pclk1 <= pclk1_max);
 
         let pclk2 = self.pclk2.unwrap_or(core::cmp::min(pclk2_max, hclk));
-        let (ppre2_bits, ppre2) = match hclk / pclk2 {
+        let (ppre2_bits, ppre2) = match (hclk + pclk2 -1) / pclk2 {
```

Comment:
To be very nitpicky, this should probably be:
```suggestion
        let (ppre2_bits, ppre2) = match (hclk + pclk2 - 1) / pclk2 {
```

Reply:
Oh, good catch. Thanks for the review. Fixed.